nncf/torch/model_creation.py

```diff
@@ -100,6 +101,11 @@ def create_compressed_model(
         is an instance of CompositeCompressionController) and the model ready for compression parameter training wrapped
         as an object of NNCFNetwork.
     """
+
+    warning_deprecated(
+        "The 'nncf.torch.create_compressed_model' function is deprecated and will be removed in a future release."
```

Comment:
The deprecation message includes a recommendation of what to use instead. Let's add it.

Reply:
```
The 'nncf.torch.create_compressed_model' function is deprecated and will be removed in a future release.
For quantization use 'nncf.quantize', another algorithms are deprecated.
```
ok?

Reply:
@AlexKoff88 , we deprecate API here but we have to state if there is an alternative recommended way. For the quantization, it's nncf.quantize() but there is no one for other algorithms. Are you ok if we state them as deprecated?

Reply:
It is a good idea and we can say something like "For QAT, use the new nncf.quantize() API from the example ...". I would omit to mention other algorithms as there is no clarity of what we will keep eventually.

Reply:
@AlexanderDokuchaev , please update the warning and then we will merge the PR.

Reply:
Updated